protokt-codegen/src/test/resources/non_null_optional.proto

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (c) 2022 Toast Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+syntax = "proto2";
```

Comment:
Interestingly, this one has to be `proto2` since for `proto3` the plugin is queried to see if it supports `optional`, and our "plugin" is just `exec cat` so it doesn't respond.